users/albertia/index.md

```diff
@@ -8,29 +8,27 @@ author: Tony Alberti
 
-Since then, I have been pursuing my Doctoral degree in *a priori* parametric model reduction. In spite of progress made in mathematical methods and computer science, many problems in Nuclear Engineering suffer from what is known as the "curse of dimensionality".
+Since then, I have been pursuing my Doctoral degree in *a priori* parametric model reduction. High fidelity (i.e. aiming for high accuracy) simulations in nuclear reactor physics inherently lie in high-parameter spaces and commonly suffer from what is known as the "curse of dimensionality". In spite of the many advances in mathematics and computer science, many of these problems are even too costly for the largest supercomputers at our national labs.
 
-One relevant example is the solution of a nuclear reactor dynamics problem with deterministic treatments of each of the phase space variables. The solution, neutron population as a function of time, has 7 unknowns; 3 in space, 2 in direction of flight, 1 in energy, and 1 in time. Couple that with a coarse mesh of say, $$ 10^3 $$, and the solution has $$ 10^{21} $$ degrees of freedom. 
+Due to the combination of high parametric space and [increasing need for high-fidelity calculations](https://energy.gov/ne/nuclear-reactor-technologies/advanced-modeling-simulation), there has been a developing interest in applications of contemporary reduced order modeling methods. Unfortunately, as one can imagine, obtaining high quality experimental or simulation data for a nuclear reactor can be difficult and costly; thus making the use of more developed data-driven methods (such as proper orthogonal decomposition or active subspaces) infeasible.
 
-You can find a short, two page abstract of this work [here](./files/SIAM_DR17_Abstract_Alberti.pdf).
-
-I have interned at Idaho National Laboratory (INL) in Idaho Falls, ID in the summer of 2015 and 2016. At INL I worked with the Rattlesnake/MAMMOTH Reactor Physics and Analysis group, specifically under the guidance of Javier Ortensi.
+Rather, my work focuses on adapting what is known as the _proper generalized decomposition_ for transient neutron diffusion. Rather than get into the details here, you can find a short, three page abstract of this work [here](./files/Alberti_ICTT2017_Abstract.pdf). If you are interested in my work or just have general questions about nuclear related things, please contact me!
 
 You can reach me via email at albertia@oregonstate.edu  
 [Resume](./files/Alberti_Resume_Public.pdf)
 
 ### Research Interests
-1.  Reactor analysis and methods
-2.  Reduced order modeling
-3.  Nuclear fuels analysis
-4.  Optimization techniques
+1. Computational Radiation Transport
+2. Reduced order modeling
+3. Nuclear reactor analysis
+4. Nuclear fuels analysis
 
 ### Conference Attendance
```

Comment:
Personally, I'd break this up into past conferences and stuff you're planning to / have been accepted to, for clarity

Reply:
Maybe if I go to more conferences and have more than just three to list I will do this! I feel like it might be a little silly to have one conference under one subsection and two under another. Hopefully a year from now, I'll have enough attendance and I'll do this!